### PR TITLE
Chore/merge-v4-dev@6fd2030

### PR DESCRIPTION
**Pull request by julien-deramond**
- [x] https://github.com/twbs/bootstrap/commit/5968ef23ca544f138c204d4e1cdd86a8f9cca247
- [x] https://github.com/twbs/bootstrap/commit/4444f090b6c6eda52a232fd75e5f3a336a171404
- [x] https://github.com/twbs/bootstrap/commit/8274bf25d062c88ea91ab3e698457856ef4db60a
- [x] https://github.com/twbs/bootstrap/commit/4e951f8bc9cf3d0f60dd685027632f245022546a
- [x] https://github.com/twbs/bootstrap/commit/b4c994d2960ca364d04764bd0787c24006f0200b
- [x] https://github.com/twbs/bootstrap/commit/d136302a5710e61f36b5370e4e608d5582030f68
- [x] https://github.com/twbs/bootstrap/commit/6fd203060baeb2c49bd4bb3f83a2837762802f95

### CODE REVIEW

File: site/content/docs/4.6/getting-started/accessibility.md [hunk 1]
```diff
@@ -24,12 +24,12 @@ Because Boosted's components are purposely designed to be fairly generic, author
 
 ### Color contrast
 
-<!-- Boosted mod -->
-Some colors that currently make up [Boosted's default palette]({{< docsref "/about/brand#color-palette" >}}) —used throughout the framework for things such as button variations, alert variations, form validation indicators— may lead to *insufficient* color contrast.
+Some combinations of colors that currently make up [Boosted's default palette]({{< docsref "/about/brand#color-palette" >}})—used throughout the framework for things such as button variations, alert variations, form validation indicators—may lead to *insufficient* color contrast (below the recommended [WCAG 2.1 text color contrast ratio of 4.5:1](https://www.w3.org/TR/WCAG/#contrast-minimum) and the [WCAG 2.1 non-text color contrast ratio of 3:1](https://www.w3.org/TR/WCAG/#non-text-contrast)), particularly when used against a light background. Authors are encouraged to test their specific uses of color and, where necessary, manually modify/extend these default colors to ensure adequate color contrast ratios.
```
Comment (julien-deramond) [inline site/content/docs/4.6/getting-started/accessibility.md:27]: I've made some modifications in this file in order to be closer to what's explained in Bootstrap while keeping the spirit of those sections. Please check if this content stays OK.

File: site/layouts/partials/docs-navbar.html [hunk 2]
```diff
@@ -34,12 +34,12 @@
           <a class="nav-link{{ if eq .Page.Title "Boostwatch" }} active{{ end }}" href="/docs/{{ .Site.Params.docs_version }}/boostwatch/" {{ if eq .Page.Title "Boostwatch" }}aria-current="page"{{ end }} onclick="dataLayer.push({'event': 'clic', 'site_name':'accessibility-boosted', 'phase':'prod', 'track_category':'navbar', 'track_name':'boostwatch'});">Boostwatch</a>
         </li>
       </ul>
-      <ul class="navbar-nav flex-row ml-md-auto d-none d-md-flex">
```
Comment (julien-deramond) [inline site/layouts/partials/docs-navbar.html:37]: It is now possible to select the version of the docs under md breakpoints. Dropdown is a bit ugly but at least this feature is now accessible.

File: site/content/docs/4.6/getting-started/accessibility.md (hunk 1 above)
Comment (louismaximepiton) [reply site/content/docs/4.6/getting-started/accessibility.md:27]: The content is good for me. Just a typo question: is that `indicators—may...` or `indicators— may...` ?

File: site/layouts/partials/docs-navbar.html (hunk 2 above)
Comment (louismaximepiton) [reply site/layouts/partials/docs-navbar.html:37]: This is a great feature ! 

However, I have 2 points here: 
- the version button is over the dropdown border
- missing border between last item and version button/github button
image

**Review by louismaximepiton — CHANGES_REQUESTED** [review]
Some minor changes to bring before merge.

File: site/content/docs/4.6/getting-started/accessibility.md (hunk 1 above)
Comment (julien-deramond) [reply site/content/docs/4.6/getting-started/accessibility.md:27]: `indicators-may`

File: site/layouts/partials/docs-navbar.html (hunk 2 above)
Comment (julien-deramond) [reply site/layouts/partials/docs-navbar.html:37]: Fixed some details via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1304/commits/2f7630191dd34b4c0340f78c37ac167b462008bc

**Review by louismaximepiton — APPROVED** [review]
LGTM !

**Pull request merged**